hyperlink/test/test_scheme_registration.py

```diff
@@ -0,0 +1,51 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+
+from .common import HyperlinkTestCase
+from .._url import register_scheme, URL
+
+
+class TestSchemeRegistration(HyperlinkTestCase):
+
+    def test_register_scheme_basic(self):
+        register_scheme('deltron', uses_netloc=True, default_port=3030)
```

Comment:
Defaulting to `uses_netloc=True` puts you in the uncomfortable position of having a default signature whose invocation raises an exception, because `default_port=None`.  If that seems OK, I'm OK with this.

But why not just make `uses_netloc` a required argument?  Then `default_port` can remain `None`, because if the user chooses to do `register_scheme('deltron', uses_netloc=False)`, they won't encounter an exception.